assets/js/src/listing/notices.jsx

```diff
@@ -0,0 +1,111 @@
+import MailPoet from 'mailpoet';
+import PropTypes from 'prop-types';
+import React, { Fragment } from 'react';
+import ReactStringReplace from 'react-string-replace';
+
+const resumeMailerSending = () => {
+  MailPoet.Ajax.post({
+    api_version: window.mailpoet_api_version,
+    endpoint: 'mailer',
+    action: 'resumeSending',
+  }).done(() => {
+    MailPoet.Notice.success(MailPoet.I18n.t('mailerSendingResumedNotice'));
```

Comment:
Above this line there was a `MailPoet.Notice.hide('mailpoet_mailer_error');`. Why did it dissapear? Maybe there's a reason for that but you removed it in a single commit with a huge move so it's extremely hard to even spot this. And since it's not a separate commit I have no idea if it's an error or not.

Reply:
I removed it because we don't use `MailPoet.Notice` to render the `mailpoet_mailer_error` anymore.
The notice will be hidden (not rendered) by React. The next line calls `window.mailpoet_listing.forceUpdate()` which will rerender the `Listing` component which will rerender `MailerError` which will return `null` because the sending was resumed.